packages/babel-helper-evaluate-path/src/index.js

```diff
@@ -8,3 +41,172 @@ module.exports = function evaluate(path) {
     };
   }
 };
+
+// Original Source:
+// https://github.com/babel/babel/blob/master/packages/babel-traverse/src/path/evaluation.js
+// modified for Babili use
+function evaluateIdentifier(path) {
+  if (!path.isReferencedIdentifier()) {
```

Comment:
This will help in catching bugs where the refId path is replaced with something else. This has been an issue in DCE+mangle previously.